interface/wc_treasury.gfx

```diff
@@ -487,4 +487,19 @@ spriteTypes = {
 		name = "GFX_bloodhoof_runespear_small"
 		texturefile = "gfx\\interface\\inventory\\artifacts\\bloodhoof_runespear_small.dds"
 	}
+	spriteType = {
+		name = "GFX_twin_blades_of_azzinoth"
+		texturefile = "gfx\\interface\\inventory\\artifacts\\twin_blades_of_azzinoth.dds"
+	}
+	spriteType = {
+		name = "GFX_twin_blades_of_azzinoth_small"
+		texturefile = "gfx\\interface\\inventory\\artifacts\\twin_blades_of_azzinoth_small.dds"
```

Comment:
This .dds image doesn't exist. So, the small icon doesn't work.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/70865362-24124300-1f76-11ea-98ab-f848532be37c.png)

</details>